openfisca_france/model/prestations/education.py

```diff
@@ -310,6 +310,19 @@ class scolarite(Variable):
     set_input = set_input_dispatch_by_period
 
 
+class boursier(Variable):
+    value_type = bool
+    entity = Individu
+    label = "Élève ou étudiant boursier"
+    definition_period = MONTH
+
+    def formula_2021(individu, period):
+        college = individu.famille('bourse_college_echelon', period)
+        lycee = individu.famille('bourse_lycee', period)
+        sup = individu('bourse_criteres_sociaux', period)
+        return (college > 0) + (lycee > 0) + (sup > 0)
```

Comment:
Nous pouvons en profiter pour améliorer la relecture de ce calcul :
```suggestion
        boursier_college = individu.famille('bourse_college_echelon', period) > 0
        boursier_lycee = individu.famille('bourse_lycee', period) > 0
        boursier_sup = individu('bourse_criteres_sociaux', period) > 0
        return boursier_college + boursier_lycee + boursier_sup
```